src/components/Calculator.js

```diff
@@ -0,0 +1,85 @@
+// /src/components/Calculator.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  const { value } = props;
+  let classes4btn = 'calcBtn';
+  if (value === '0') {
+    classes4btn = 'calcBtn span2';
+  }
+  if (value === '+' || value === 'x' || value === '-' || value === '=') {
+    classes4btn = 'calcBtn orangeBtn';
+  }
```

Comment:
* Kindly use class-based components for all your components as stated in the project requirement [Check here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).)